.github/workflows/sync_github_project.yml

```
@@ -0,0 +1,240 @@
+            for (const num of numbers) {
+              const issue = await github.rest.issues.get({ owner, repo, issue_number: num });
+              const nodeId = issue.data.node_id;
+              await github.graphql(
+                `mutation($projectId:ID!,$contentId:ID!){ addProjectV2ItemById(input:{ projectId:$projectId, contentId:$contentId }){ item { id } } }`,
+                { projectId: projectNodeId, contentId: nodeId }
+              );
+              core.info(`Added #${num} to project`);
+            }
+
+      - name: Update Project Status (optional)
+        if: ${{ secrets.PROJECT_URL != '' }}
```

actionlint (1.7.7)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)